pallets/runtime/tests/mod.rs

```diff
@@ -1,11 +1,6 @@
-#![allow(dead_code)]
-
-pub mod storage;
-pub use storage::{
+mod all;
```

Comment:
Is the idea behind this move to separate tests from utility functions? Why instead not to have utility functions moved to a subdirectory?

Reply:
The previous structure generated individual binary for every test file. This new structure generates a single binary for the all the tests.

Generating individual binaries takes more compilation time and space. With this change, the time to run tests in CI has been cut from 26 min to 8 min.

There's one disadvantage of this change though. In the previous structure, you could have changed one test file and only recompiled that but now you have to recompile all of them every time. That being said, compiling an individual test file took ~5 seconds while compiling all of them takes ~15 seconds and hence I think the trade-off is worth it. ps If you change the actual code (outside the tests) then the wasm runtime has to be recompiled which takes ~4 minutes and makes this 10sec gain irrelevant.

Given the trade-off, I think this change is worth doing.